README.md

```diff
@@ -1 +1,23 @@
 # Public talks by Hotels.com Technology
+In this repo you can find presentations given by
+[Hotels.com Technology](https://medium.com/hotels-com-technology) (part of Expedia Group) at conferences,
+meetups, tech events, and more. When available links to recordings are
+provided.
+
+Having a list of our presentations on GitHub has the following benefits:
+- There's a single place where people can find our presentations, and
+  even helps us keep track of our external talks.
+- It makes it easier for people to get an idea of the areas we're
+  working on or are interested in. Of course, this doesn't mean that
+  these are the only areas Hotels.com Technology has an interest in.
+
+## 2019
+
+| Date | Event | Presentation Title | Topic(s) | Video/Slides |
+|:-----|:------|:-------------------|:---------|:-------------|
+
+## 2018
+| Date       | Event                    | Presentation Title                                                  | Topic(s)                            | Video/Slides                                                                                                                                                                    |
+|:-----------|:-------------------------|:--------------------------------------------------------------------|:------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 2018-11-20 | Voxxed Days Thessaloniki | Improving your team’s source code searching capabilities            | Dev Tools                           | [Video](https://youtu.be/QDAY4o03t2c) - [Slides](https://www.slideshare.net/NikolaosKatirtzis/improving-your-teams-source-code-searching-capabilities-voxxed-thessaloniki-2018) |
```

Comment:
Can you make the "Events" clickable to the webpage of the event itself?

Reply:
The problem with links to conference pages is that usually these go down after some time and are no more accessible. 
Unless we verify this occasionally this would be problematic.

Reply:
OK, I guess it depends on the conference, some have more stable URLs than others, we can leave this up to the discretion of whoever is adding the entry.

Reply:
Yeah true. I ended up adding links for these 2 presentations cause these look stable. Thanks for pointing out.